examples/data/assessments.csv

```diff
@@ -0,0 +1,5 @@
+id,name

```

Comment:
@mjoss17 - Can you help confirm that these CSV files are correct / reflect the latest? I copied them blindly from [this Google Drive folder](https://drive.google.com/drive/folders/1O2CX53ZXPv68HQyK_DWLXFy3g27SWRO3) :sweat_smile: .

Reply:
Everything checks out